tools/gosec/gosec.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"encoding/xml"
+	"flag"
+	"io/ioutil"
+	"log"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// These structs are copied from gosec
```

Comment:
Can't we rather vendor in the dependencies? Or why do we copy them here?

Reply:
The only reason is that they are private.